docker-compose.yml

```diff
@@ -2,16 +2,18 @@
 version: '3.7'
 services:
   pmm-managed-server:
-    build:
-      context: .
-      args:
-        PMM_VERSION: ${PMM_VERSION:-perconalab/pmm-server:dev-latest}
-      dockerfile: devcontainer.Dockerfile
+    image: perconalab/pmm-server:dev-container
+#    build:
+#      context: .
+#      args:
+#        PMM_VERSION: ${PMM_VERSION:-perconalab/pmm-server:dev-latest}
+#      dockerfile: devcontainer.Dockerfile
     container_name: pmm-managed-server
     hostname: pmm-managed-server
     networks:
       - ${NETWORK:-default}
     environment:
+      - PMM_RELEASE_PATH=/root/go/bin
```

Comment:
I guess we can remove it, `PMM_RELEASE_PATH` has a default value defined in makefile

Reply:
I think the comment below explains why we need to override PMM_RELEASE_PATH here

Reply:
@BupycHuk yes, you are correct. It is a nasty bug I was able to reproduce after a while on Linux, on mac all works.

Reply:
Then I didn't get it. Makefile defines it's own `PMM_RELEASE_PATH` and all actions we perform with makefile targets. What it the reason to overwrite it?